src/backend/src/common/entities/base.entity.ts

```diff
@@ -38,4 +38,19 @@ export class BaseEntity {
     name: 'created_by_username',
   })
   createdByUsername: string;
+
+  @Column({
+    name: 'updated_by_guid',
+  })
+  updatedByGuid: string;
+
+  @Column({
+    name: 'updated_by_username',
+  })
+  updatedByUsername: string;
+
+  @Column({
+    name: 'save_id',
+  })
```

Comment:
```suggestion
  @Column({
    name: 'save_id',
    nullable: false,
  })
```

Reply:
Also, adding it to the base_entity will add to all the tables. should be inside pia-intake.entity.ts